server/src/main/java/org/tctalent/server/service/db/impl/SavedSearchServiceImpl.java

```diff
@@ -361,29 +361,32 @@ public Page<Candidate> searchCandidates(SearchCandidateRequest request) {
         Page<Candidate> candidates;
         User user = userService.getLoggedInUser();
         if (user == null) {
+            // todo when is the user null?
```

Comment:
I'm not quite sure when this would be the case - did give it a proper dig around for an answer but thought you might remember?